dmscripts/helpers/logging_helpers.py

```diff
@@ -1,5 +1,3 @@
-from __future__ import absolute_import
```

Comment:
Same as the `u''`. I'm not really sure what these are for in this case, but they make no difference for Python 3 so we might as well keep them until we're sure we don't need it.